plugins/utils/auto-save.nix

```
@@ -6,65 +6,94 @@
   ...
 }:
 with lib;
-let
-  cfg = config.plugins.auto-save;
-in
-{
-  options.plugins.auto-save = helpers.neovim-plugin.extraOptionsOptions // {
-    enable = mkEnableOption "auto-save";
-
-    package = helpers.mkPluginPackageOption "auto-save" pkgs.vimPlugins.auto-save-nvim;
-
-    keymaps = {
-      silent = mkOption {
-        type = types.bool;
-        description = "Whether auto-save keymaps should be silent.";
-        default = false;
-      };
-
-      toggle = helpers.mkNullOrOption types.str "Keymap for running auto-save.";
-    };
-
-    enableAutoSave = helpers.defaultNullOpts.mkBool true ''
-      Whether to start auto-save when the plugin is loaded.
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "auto-save";
+  originalName = "auto-save.nvim";
+  defaultPackage = pkgs.vimPlugins.auto-save-nvim;
+
+  maintainers = [ maintainers.braindefender ];
```

```suggestion
  maintainers = [ helpers.maintainers.braindefender ];
```

Also, could you move adding yourself to the list to a separate commit, titled "maintainers: add braindefender"